api/openapi.yaml

```diff
@@ -2378,7 +2378,7 @@ components:
               $ref: '#/components/schemas/ScanRelationship'
             status:
               $ref: '#/components/schemas/AssetScanStatus'
-            sboms:
+            sbom:
```

Comment:
I renamed because it is called `sbom` here as well: https://github.com/openclarity/vmclarity/blob/main/api/openapi.yaml#L2426
It would complicate the code a lot if we had to handle it as an exception on the frontend when mapping through the items.